src/wrappers/msvc_wrapper.cpp

```diff
@@ -37,6 +37,11 @@ bool is_source_file(const std::string& arg) {
   return ((ext == ".cpp") || (ext == ".cc") || (ext == ".cxx") || (ext == ".c"));
 }
 
+bool is_object_file(const std::string& file_ext) {
+  const auto ext = lower_case(file_ext);
+  return ((ext == ".obj") || (ext == ".o"));
```

Comment:
https://docs.microsoft.com/en-us/cpp/build/reference/fo-object-file-name?view=vs-2019 makes me wonder if we need to check an extension at all?  We could possibly just check if the /Fo argument doesn't end with a slash.  But this was the smallest/safest change.